<launch>
    <include file="$(find amigo_machine_files)/machines/amigo4.machine" />

    <node pkg="ompl_ros_interface" type="ompl_ros" name="ompl_planning" machine="amigo4">
        <param name="default_planner_config" type="string" value="SBLkConfig1" />
        <rosparam command="load" file="$(find amigo_arm_navigation)/config/ompl_planning.yaml" />
    </node>

</launch>
